packages/eslint-plugin-ckeditor5-rules/lib/rules/require-file-extensions-in-imports.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { extname, dirname, sep, parse } = require( 'path' );
```

Comment:
```suggestion
const { extname, dirname, sep, parse } = require( 'upath' );
```

We recommend using `upath` due to issues with a directory separator across different OSes.